src/server/middleware/sendHtml.js

```diff
@@ -40,8 +40,8 @@ function getChunkAssets(assetsByChunkName) {
     .map(([, assets]) => (typeof assets === 'string' ? assets : assets[0]));
 }
 
-const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.browser.json').assetsByChunkName);
-const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.legacyBrowser.json').assetsByChunkName)
+const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').modernBrowserChunkAssets);
+const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').legacyBrowserChunkAssets)
```

Comment:
```suggestion
const { modernBrowserChunkAssets } = getChunkAssets(readJsonFile('../../../.build-meta.json'));
const { legacyBrowserChunkAssets } = getChunkAssets(readJsonFile('../../../.build-meta.json'));
```
destructuring might be a style change though?

Reply:
oh, would be better to combine:
```suggestion
const { modernBrowserChunkAssets, legacyBrowserChunkAssets } = getChunkAssets(readJsonFile('../../../.build-meta.json'));
```

Reply:
See https://github.com/americanexpress/one-app/pull/22#discussion_r373794000